core-bundle/tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -292,7 +292,7 @@ private function checkKeys(array $configuration): void
                 $this->checkKeys($prototype->getChildren());
             }
 
-            if (\is_string($key) && !$value->isDeprecated()) {
+            if (\is_string($key) && !$value->isDeprecated() && 'resampling-filter' !== $key) {
```

Comment:
See https://github.com/contao/image/pull/108#discussion_r1784113131
I think we should use the original name as it is used by Imagine, WDYT?

Reply:
I agree.